modules/ocf_mail/files/site_ocf/update-access

```diff
@@ -1,11 +1,21 @@
-#!/bin/bash -eu
-# Update addresses that are allowed to send mail to protected addresses.
+#!/bin/bash
+# Update addresses that are allowed to send to protected addresses. Those
+# being OCF addresses and corresponding LDAP forwarding addresses of staff
+set -euo pipefail
 
-set -o pipefail
+TMP=$(mktemp /tmp/staff-access.XXXXX)
 
-# staff
-getent group ocfstaff | cut -d: -f4 | \
-	sed 's/,\|$/@ocf.berkeley.edu OK\n/g' \
-	> /etc/postfix/ocf/staff-access
+# Load the keytab so we can pull the mail attr from LDAP
+kinit -t /etc/postfix/ocf/smtp-krb5.keytab smtp/anthrax.ocf.berkeley.edu
+
+USERNAMES="$(getent group ocfstaff | cut -d: -f4 | tr ',' '\n')"
+
+while read -r user; do
```

Comment:
imo we should just do `for user in $USERNAMES`. This also lets us `tr` with spaces instead of newlines which I find slightly simpler.